docs/casa/manual/chapters/30_panels_and_clips.md

```diff
@@ -0,0 +1,53 @@
+# Panels and Clips
+
+![](../img/30_1.png)
+
+---
+
+Begin by fitting the static panels to the frame. The panels fit over the extrusion and are fixed in place using printed clips. Each panel should have weather stripping applied to all edges.
```

Comment:
Maybe we should specify all inner edges?

Reply:
added